### PR TITLE
Static page

### PR DESCRIPTION
**Pull request by chatandmary**

### CODE REVIEW

File: app/assets/stylesheets/custom.scss [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+h1, h2, h3, p,h4, h5, ul, ol, li, dl, dt, dd, form, figure, form {
+  margin: 0px;
+  padding: 0px;
+  font-size: 100%;
+  font-weight: normal;
+  line-height: 2;
+}
+
+.clear {
+clear:both;
```
Comment (ivanraybuglosa) [inline app/assets/stylesheets/custom.scss]: Please add indention to this property

File: app/views/layouts/_header.html.erb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<header>
+        <div class="hl container header-title">
+          <%= link_to("Cannabis", root_path , class:"eden")%>
+            <nav class="hr">
+              <ul class="list-inline">
+                <li class="list-inline-item"><%= link_to "ABOUT", about_path, id:"about"%></li>
+                <li class="list-inline-item"><a href="#" id="signup">SIGNUP</a></li>
```
Comment (ivanraybuglosa) [inline app/views/layouts/_header.html.erb]: Please change anchor tag to link_to

File: app/views/static_pages/home.html.erb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+          <div class="row top-row">
+            <div class="col-md-6 col-12 top-col-left order-2 order-md-1">
+                <h2 class="description">
+                Let's<br class="br-n">
+                Learning<br class="br-n">
+                About
+                Cannabis :)
+                </h2>
+                <div class="button">
+                <%= link_to('Member Registration', "#",class:"reg")%>
```
Comment (ivanraybuglosa) [inline app/views/static_pages/home.html.erb]: Please indention

File: app/assets/stylesheets/custom.scss (hunk 1 above)
Comment (chatandmary) [reply app/assets/stylesheets/custom.scss]: I fixed indention properly

**Pull request merged**